client/cypress/integration/nav-bar.spec.js

```diff
@@ -0,0 +1,34 @@
+describe("Navigation bar", () => {
+  it("shows page title", () => {
+    cy.visit("/");
+    cy.get("#nav-bar").get("#title").contains("OpenTutor");
+    cy.visit("/lessons");
+    cy.get("#nav-bar").get("#title").contains("Lessons");
+    cy.visit("/lessons/edit?lessonId=new");
+    cy.get("#nav-bar").get("#title").contains("Edit Lesson");
+    cy.visit("/sessions");
+    cy.get("#nav-bar").get("#title").contains("Grading");
+    cy.visit("/sessions/session");
+    cy.get("#nav-bar").get("#title").contains("Grade Session");
+  });
+
+  it("opens drawer menu", () => {
+    cy.visit("/");
+    cy.get("#drawer").should("not.exist");
+    cy.get("#nav-bar").get("#menu-button").click();
+    cy.get("#drawer");
```

Comment:
would this fail if the drawer didn't open?

Maybe this type of test is a good candidate for `matchImageSnapshot`?

Reply:
Yeah, it would because #drawer wouldn't exist in the DOM